### ISSUE TITLE
LLM Embedder FT

### ISSUE THREAD

**Issue by chiosChen** [FlagOpen/FlagEmbedding#174]
Is the teacher score in training data same as the reward from LLM in the paper? 

How do I generate those scores for my own data?

**Comment by namespace-Pt** [comment]
Hi,

Yes, they are generated by the LLM. You can refer to the *LM Scoring* section in [this documentation](https://github.com/FlagOpen/FlagEmbedding/blob/master/FlagEmbedding/llm_embedder/docs/fine-tune.md). 

Note that you should follow the data format in this documentation. For example, to get the reward of a series of positive and negative documents `["pos doc", "neg doc 1", "neg doc 2", "neg doc 3"]` w.r.t. `test query 1`, your data sample should be:

```python
{"query": "test query 1", "pos": ["pos doc"], "neg": ["neg doc 1", "neg doc 2", "neg doc 3"], "answers": ["answer to test query 1"], "task": "qa"}
```
where `answers` denote the expected output from LLM.

**Comment by chiosChen** [comment]
Great, thanks for the response
